client/src/components/screens/Patient/components/FilesTab/components/tableColumns.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import intl from 'react-intl-universal';
+import { useDispatch } from 'react-redux';
+import { DownOutlined } from '@ant-design/icons';
+import { getPatientFileURL } from 'actions/patient';
+import {
+    Button, Dropdown, Menu,
+} from 'antd';
+import { PatientFileFormat,PatientResponse, } from 'helpers/search/types';
+import { sortByOldest } from 'helpers/utils'
+
+const getDropdownOption = (
```

Comment:
extract and rename this in another component in a file of its own.

Reply:
column et dropdown devrait être séparé.